cluster/gce/config-common.sh

```diff
@@ -98,6 +98,14 @@ function get-cluster-ip-range {
   echo "${suggested_range}"
 }
 
+function get-alias-range-size() {
+  for pow in {0..8}; do
```

Comment:
Why just 0..8? We should go all the way to 31, as we are not limiting the max pod size.

Reply:
You're right I shouldn't set this limit here.